### DISCUSSION TITLE
Deeature Request: Debian packaging

### DISCUSSION THREAD

**Discussion by gcsideal** [duckdb/duckdb#10071] [Ideas]
It would help if DuckDB is available in Debian as a package. It was requested [1] and I've packaged it, but it has some issues.
It installs libraries the old way, under /usr/lib directory. It should switch to using the multi-arch paths [2] as other projects do for a while.
Then it embeds several other projects under the third_party directory, I can't find out how to prevent that and use the system wide installed ones. This would help to get and use bug and even security fixes immediately - there would be no need to incorporate those to DuckDB separately. Of course, the third_party directory can remain as a fallback for systems don't have the needed dependency installed. But it shouldn't be the first nor the only one option.
Outdated parts. All modern systems have CMake 3.5 at least, as 3.0 is released ten years ago and is available as a Debian package as well for about the same time [3]. As such asking for the minimum CMake version of 2.8.12 in extension/jemalloc/CMakeLists.txt might imply a file to update or the version check is no longer such relevant (I guess other parts of the project need a more recent version of CMake anyway).
The version of D3js in extension/visualizer/d3.js is 6.7.0 (current is v7.8.5) and from 2021.

Hope these can be addressed and a new release of DuckDB can be packaged for Debian.

[1] https://bugs.debian.org/1036922
[2] https://wiki.ubuntu.com/MultiarchSpec
[3] https://tracker.debian.org/news/570660/accepted-cmake-302-1-source-all-into-unstable/

**Comment by mocusez** [comment]
Hi, I try to pack a debian release on https://github.com/mocusez/duckdb_deb_package/releases
And the work was already upload  to Debian Mentor https://mentors.debian.net/package/duckdb
